lib/s3-adapter.js

```diff
@@ -82,12 +85,37 @@ module.exports = Adapter.extend({
 
   _printBucketRevisions: function(revisions) {
     var header = green('Found the following revisions: \n');
-
+    var _self=this;
     var revisionsList = revisions.reduce(function(prev, current, index) {
-      return prev + '\n\n' + (index + 1) + ') ' + current;
-    }, '');
+      var revisionMessage = prev + '\n' + (index + 1) + ') ' + current;
+
+      if (this.currentRevision && this.currentRevision === current) {
+        revisionMessage = red(revisionMessage);
+      }
 
-    var footer = green('\n\nUse activate() to activate one of these revisions');
+      if (_self.config.url) {
+        revisionMessage += '  (' + blue(_self.config.url + current) + ')';
+      }
+      revisionMessage += '\n';
+
+      if (_self.config.verbose) {
+        var verboseMessage;
```

Comment:
What part of the tagging adapter? I considered it, but the tagging adapter api only has a createTag function.